ui/app/projects/projects.module.js

```diff
@@ -42,6 +42,19 @@
         }
       });
 
+      var getSortFromLocalStorage = function () {
```

Comment:
this function should be refactored out, but I think it can be done separately as a part of NCL-5347

Reply:
+1, better to have an injectible factory method that can be reusued

Reply:
Will fix it in NCL-5347